src/server/api/routes/cvs.router.js

```diff
@@ -153,7 +150,7 @@ router.patch('/:id', (req, res, next) => {
  */
 router.delete('/:id', (req, res) => {
   cvsController
-    .deleteModule(req.params.id, req)
+    .deleteCv(req.params.id, req)
```

Comment:
@CharlesKimMaina  this is the only line i would have expected to be checked in. Cheers,